src/vs/workbench/contrib/scm/browser/scmViewPane.ts

```diff
@@ -1088,6 +1088,7 @@ class ViewModel {
 		this.modeContextKey.set(mode);
 
 		this.storageService.store(`scm.viewMode`, mode, StorageScope.WORKSPACE, StorageTarget.MACHINE);
+		this.storageService.store(`scm.viewMode`, mode, StorageScope.WORKSPACE, StorageTarget.USER);
```

Comment:
Why is this duplicating the previous line?

Reply:
Good point, I thought this would be a necessary backup in case the user doesn't have Cloud Changes enabled, but I see now that it's not required following some testing.